src/test/java/com/wikia/webdriver/testcases/featuredvideo/FeaturedVideoMobileTests.java

```diff
@@ -0,0 +1,94 @@
+package com.wikia.webdriver.testcases.featuredvideo;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.featuredvideo.FeaturedVideoDesktopComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.featuredvideo.FeaturedVideoMobileComponentObject;
+
+import org.testng.annotations.Test;
+
+@Test(groups = {"FeaturedVideoMobile"})
+@Execute(onWikia = "featuredvideo", asUser = User.ANONYMOUS)
+@InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+public class FeaturedVideoMobileTests extends NewTestTemplate {
+
+  @Test
+  public void videoIsPresentOnArticle() {
+    FeaturedVideoMobileComponentObject video = new FeaturedVideoMobileComponentObject()
+        .setAutoplayCookie(false)
+        .openWikiArticle("FeaturedVideo");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FeaturedVideo" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)